cirq-core/cirq/sim/simulator.py

```diff
@@ -560,11 +589,11 @@ def _create_act_on_args(
         """
         raise NotImplementedError()
 
-    @abc.abstractmethod
     def _core_iterator(
```

Comment:
[No change needed] There's some ongoing discussion around naming of "protected" methods in #3898 - nothing concrete yet, but it would be good to keep an eye on that issue.